_posts/2021-12-06-ethics-surveillance-tech.md

```diff
@@ -0,0 +1,216 @@
+---
+layout: post
+name:  "Opinion: Pseudo-ethics in the Surveillance Tech Industry"
+title:  "Opinion: Pseudo-ethics in the Surveillance Tech Industry"
+date:   2021-12-06 10:00:00 +0200
+author: circe
+published: true
+permalink: /ethics-surveillance-tech
+categories: research
+summary: A look at typical ethical shortfalls in the global surveillance tech industry.
+image: /assets/img/vac.png
+discuss:
+---
+
```

Comment:
Maybe preface with a sentence on this being an anon opinion piece? *This was written by pseudonymous...*